scipy/stats/tests/test_distributions.py

```diff
@@ -3089,6 +3089,46 @@ def test_moments_stats(self, r):
         # r-th non-central moment is non-finite (inf or nan) if r >= c.
         assert not np.any(np.isfinite(stats.loglaplace.stats(c, moments=mom)))
 
+    @pytest.mark.parametrize("c", [0.5, 1.0, 2.0])
+    @pytest.mark.parametrize("loc, scale", [(-1.2, 3.45)])
+    @pytest.mark.parametrize("fix_c", [True, False])
+    @pytest.mark.parametrize("fix_loc", [True])
```

Comment:
It sounds like we're not going to override the generic method, so maybe just set `floc` in the function?

Reply:
Sure I will update that!